baremaps-server/src/main/java/com/baremaps/server/CorsFilter.java

```diff
@@ -1,28 +1,61 @@
+/*
+ * Code inspired by resteasy CorsFilter: https://github.com/resteasy/Resteasy/blob/22984e96a2f40dc556ba7b1613422febd94f86d2/resteasy-core/src/main/java/org/jboss/resteasy/plugins/interceptors/CorsFilter.java
+ *
+ Copyright (c) Resteasy Maintainers. All rights reserved.
+ Licensed under the ASL license.
+ */
+
 package com.baremaps.server;
 
-import java.io.IOException;
 import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
 import javax.ws.rs.container.ContainerResponseContext;
 import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.Provider;
 
 @Provider
-public class CorsFilter implements ContainerResponseFilter {
+public class CorsFilter implements ContainerRequestFilter, ContainerResponseFilter {
+  
+    private final String ACCESS_CONTROL_ALLOW_ORIGIN = "Access-Control-Allow-Origin";
```

Comment:
Indentation problem.

Reply:
You mean? that it uses tabs or space or should use less space? or more space?

Reply:
There are some instructions in the [contributing guidelines](https://github.com/baremaps/baremaps/blob/main/CONTRIBUTING.md#providing-patches) to setup IntelliJ with google's codestyle.xml file copied at the root of the repository. Google's code style uses two spaces instead of the usual four spaces. 

https://github.com/baremaps/baremaps/blob/main/CONTRIBUTING.md#providing-patches

Reply:
Addressed in https://github.com/baremaps/baremaps/pull/319/commits/674d9a2d033c76335d099b4364f7a2281866a68d